prow/prowjobs/GoogleCloudPlatform/gcp-guest/compute-image-tools.yaml

```diff
@@ -1,5 +1,22 @@
 presubmits:
   GoogleCloudPlatform/compute-image-tools:
+  - name: compute-image-tools-export-sbom
+    cluster: gcp-guest
+    run_if_changed: 'daisy_workflows/export/export_disk.sh'
+    trigger: "(?m)^/flake8$"
```

Comment:
you need a unique trigger; this should usually match the context name.

Reply:
changed the trigger